pyodide-build/pyodide_build/tests/test_pywasmcross.py

```diff
@@ -145,6 +147,18 @@ def test_handle_command_ldflags(build_args):
     )
 
 
+@pytest.mark.parametrize(
+    "arg, expected",
+    [
+        ("-I/usr/include", None),
+        (f"-I{sys.prefix}/include/python3.11", "-I/target/include/python3.11"),
```

Comment:
Could we money patch base_prefix and prefix for the test? Or do you think that's overkill?

Reply:
I think it is a good idea. I'll add update a test